src/setupTest.js

```diff
@@ -1,8 +1,14 @@
 /* eslint-disable import/no-extraneous-dependencies */
+import 'babel-polyfill';
```

Comment:
@davidjoy is @babel/polyfill the one we should be using now? https://babeljs.io/docs/en/babel-polyfill